pint_server/models.py

```diff
@@ -104,6 +104,7 @@ class GoogleImagesModel(Base, ProviderImageBase):
 class MicrosoftImagesModel(Base, ProviderImageBase):
     __tablename__ = 'microsoftimages'
 
+    publishedon = Column(Date)
     environment = Column(String(50), primary_key=True)
```

Comment:
This will assume an autogenerated PK name and thus doesn't necessarily match the migrations. We would need something like:

```
PrimaryKeyConstraint('name', 'environment', name='microsoftimages_pkey')
```

Reply:
Good point! I will make the change once everyone agree on the proposed primary keys in the email.

Reply:
Latest patchset has the proposed changes for the image tables.